Include airoboros-33b/65b gpt4 1.2 models

jondurbin opened the pull request:
Model outputs, and GPT4 eval results here:
https://huggingface.co/datasets/jondurbin/airoboros-gpt4-1.2-alpaca-eval/tree/main

YannDubs commented:
Hi Jon!

Sorry for the misunderstanding but the right leaderboard to update is [this one](https://github.com/tatsu-lab/alpaca_eval/blob/main/src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv). I updated the doc to make it more clear. If you run 

```bash
alpaca_eval evaluate_from_model \
  --model_configs ... \
  --annotators_config 'alpaca_eval_gpt4' \
  --is_cache_leaderboard True 
```

it will update the correct leaderboard and then you just need to commit that file. This ensures that the model gets on the [main page](https://tatsu-lab.github.io/alpaca_eval/)

jondurbin commented:
Sorry about that, thanks for the update.  I've updated the branch with the correct leaderboard and added the raw outputs.

I didn't originally include the raw outputs because it seems most are not included, so let me know what your preference would be.

rtaori commented on README.md:
```diff
@@ -162,6 +162,8 @@ models that are available out of the box.
 | chatgpt               |     86.1 |       1.2 |
 | wizardlm-13b          |     75.3 |       1.5 |
+| airoboros-65b         |     73.9 |       1.5 |
+| airoboros-33b         |     73.3 |       1.6 |
```
We're trying to keep the leaderboard readme small here, so only displaying results for the `minimal` model set. Could you please revert this change?

rtaori commented:
Looks great! Ready to merge with the small readme fix :)
